CityPuzzle/Side Pages/Web Service/Constants.cs

```diff
@@ -6,6 +6,6 @@ namespace WebServiceExample
 {
     public class Constants
     {
-        public const string GitHubReposEndPoint = "https://api.github.com/orgs/dotnet/repos";
+        public const string GitHubReposEndPoint = "https://github.com/AllBill127/Skestam_Kartu/blob/WebService/CityPuzzleAPI?fbclid=IwAR3DDhRhykjnMS3YbLlgblTlLAvnK3iOF10VXdLgf59Y7DLfjcqJnVlfdPI";
```

Comment:
What exactly is this link and why is it so long, because previous link just points to /repos directory